hub/config/src/main/java/uk/gov/ida/hub/config/resources/CountriesResource.java

```diff
@@ -18,21 +17,16 @@
 public class CountriesResource {
 
     private final LocalConfigRepository<CountryConfig> countryConfigRepository;
-    private final ExceptionFactory exceptionFactory;
 
     @Inject
     public CountriesResource(
-            LocalConfigRepository<CountryConfig> countryConfigRepository,
-            ExceptionFactory exceptionFactory
```

Comment:
I think this change should be a separate commit

Reply:
`exceptionFactory` wasn't used in this class, hence removed. I really don't want to raise a separate PR for this given that it takes 2 days to get reviews

Reply:
Agree, these things don't need separate PR's, but separate commits means the intention is more clear to anyone looking at the history, and (sometimes) we have to back changes out, which makes separating stuff cleanly quite important, I think.

Reply:
ugh yeah separate commits is fair enough